### PR TITLE
Docsite: add Community guide

### PR DESCRIPTION
**Pull request by Andersson007**
Dear maintainers,
As a part of the [Consolidating Ansible discussion platforms initiative](https://forum.ansible.com/t/proposal-consolidating-ansible-discussion-platforms/6812), this PR adds the communication section template defined by the community to the README and the docsite. Similar PRs are now being raised across all the Ansible ecosystem projects.

Thank you

### CODE REVIEW

**Comment by Andersson007** [comment]
ready for review

**Comment by felixfontein** [comment]
Thanks for your contribution! I think here similar comments apply as in https://github.com/ansible-community/antsibull-changelog/pull/177#pullrequestreview-2275087297.

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing! All done, PTAL

**Comment by Andersson007** [comment]
@felixfontein I applied your suggestions from [the other PR](https://github.com/ansible-community/antsibull-changelog/pull/177) here as well, fyi, PTAL

File: docs/index.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ This package provides tooling for validating and building Ansible documentation.
 
 antsibull-docs is covered by the [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
 
+> Need help or want to discuss the project? See our [Community guide](community.md) to learn how to join the conversation!
+
```
Comment (samccann) [inline docs/index.md:19]: Similar comment from the other PR - remove the > if you don't want this to look something like the following:
image

File: docs/index.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ This package provides tooling for validating and building Ansible documentation.
 
 antsibull-docs is covered by the [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
 
+> Need help or want to discuss the project? See our [Community guide](community.md) to learn how to join the conversation!
```
Comment (felixfontein) [inline docs/index.md:18]:
```suggestion
!!! note
    Need help or want to discuss the project? See our [Community guide](community.md) to learn how to join the conversation!
```

**Review by felixfontein — APPROVED** [review]

**Comment by Andersson007** [comment]
@felixfontein @samccann thanks for reviewing!

**Pull request merged**